README.md

```diff
@@ -28,6 +28,14 @@ The former should be used for direct access to the TPM through the Linux kernel
 The latter implements the protocol exposed by the Microsoft software TPM2 simulator.
 * The [TCG TSS 2.0 Overview and Common Structures Specification](https://trustedcomputinggroup.org/wp-content/uploads/TSS_Overview_Common_Structures_Version-0.9_Revision-03_Review_030918.pdf) forms the basis for all implementations in this project.
 
+# TCTI Defaults
+If the TCTI is not specified when calling this library, the TCTI defaults to the first found in this order:
+* Library libtss2-tcti-default.so (link to the preferred TCTI)
+* Library libtss2-tcti-tabrmd.so (tabrmd)
+* Device /dev/tpmrm0 (kernel resident resource manager)
+* Device /dev/tpm0 (hardware TPM)
+* TCP socket localhost:2321 (TPM simulator)
```

Comment:
I don't think this description is not quite accurate. What do you mean by "calling this library"?
TSS SAPI exposes two init functions for TCTI:
Tss2_Tcti_Device_Init() and Tss2_Tcti_Mssim_Init()
The user of the SAPI needs to explicitly call the correct function depending on which TPM target they want to talk to.  
On top of that the ESAPI exposes its own Esys_Initialize() function, which takes TSS2_TCTI_CONTEXT *
and if this is NULL it tries to do some magic to figure out what TCTI to use.
In that case it will try to load whatever was configured via --with-tctidefaultmodule or call
Tss2_Tcti_Device_Init() and then Tss2_Tcti_Mssim_Init()